docs/help/git_webhooks.md

```diff
@@ -0,0 +1,30 @@
+## Webhooks
+
+1. Generate a SSL certificate for the REST API. TODO: figure out the right
+   way to do that.
+
+Using acme.sh on the host already, I did TODO
+
+```
+$ACMESH --key-file $BITBOTDIR/ssl.key --cert-file $BITBOTDIR/ssl.cert --reloadcmd "$SYSTEMCTLRESTART bitbot"
+chmod -R 700 $BITBOTDIR/ssl.key
+chown -R bitbot:bitbot $BITBOTDIR
+```
```

Comment:
This is part of [my script for managing acme.sh certificates](https://gitea.blesmrt.net/mikaela/scripts/src/branch/master/bash/acmesh-ssl.bash) that I am happy to expand later if you wish.

Question: what format are the keys expected in and did I even pass acme.sh the right arguments (`--key-file` and `--cert-file`)?

```
  --cert-file                       After issue/renew, the cert will be copied to this path.                                                        --key-file                        After issue/renew, the key will be copied to this path.
  --ca-file                         After issue/renew, the intermediate cert will be copied to this path.
  --fullchain-file                  After issue/renew, the fullchain cert will be copied to this path.
                                                                         
  --reloadcmd "service nginx reload" After issue/renew, it's used to reload the server.                                       
```

Reply:
Additional question, is a restart required for reloading the certificates?

Reply:
no - just reloading the sending `SIGUSR1` to bitbot will reload `bot.conf` and doing `!reloadmodule rest_api` will start the REST API with the configured cert

Reply:
:+1:

Reply:
I think it may need to be valid certificate at least for Gitea, which is worth noting and my chosen Gitea instance complains of:

`Delivery: Post https://relpda.mikaela.info:5000/api/gitea?key=: x509: certificate signed by unknown authority`

so I think I will need to give it `--fullchain-file cert.pem --key-file key.pem`